File: Java/Pegawai.java
public class Pegawai extends Manusia {
    private String idPegawai;
    private int lamaTahunBekerja;

    public Pegawai(String nama, String nohp, String idPegawai, int lamaTahunBekerja) {
        super(nama, nohp);
        this.idPegawai = idPegawai;
        this.lamaTahunBekerja = lamaTahunBekerja;
    }

    public String getIdPegawai() {
        return idPegawai;
    }

    public void setIdPegawai(String idPegawai) {
        this.idPegawai = idPegawai;
    }

    public int getLamaTahunBekerja() {
        return lamaTahunBekerja;
    }

    public void setLamaTahunBekerja(int lamaTahunBekerja) {
        this.lamaTahunBekerja = lamaTahunBekerja;
    }
}

File: Python/Pegawai.py
from Manusia import Manusia

class Pegawai(Manusia):
    __idPegawai = ""
    __LamaTahunBekerja = 0

    def __init__(self, nama="", nohp="", idPegawai="", LamaTahunBekerja=0):
        super().__init__(nama, nohp)  # Memanggil constructor dari StokProduk
        self.__idPegawai = idPegawai
        self.__LamaTahunBekerja = LamaTahunBekerja

    # Getter dan Setter
    def getidPegawai(self):
        return self.__idPegawai
    
    def setidPegawai(self, idPegawai):
        self.__idPegawai = idPegawai

    def getLamaTahunBekerja(self):
        return self.__LamaTahunBekerja
    
    def setLamaTahunBekerja(self, LamaTahunBekerja):
        self.__LamaTahunBekerja = LamaTahunBekerja
